images/cassandra/rootfs/run.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/dumb-init /bin/bash
```

Comment:
And I used it [here](https://github.com/gravitational/pithos-app/blob/master/images/cassandra/start-cron.sh#L1) before.